tests/reports/api.test.ts

```diff
@@ -11,7 +11,7 @@ describe('Reports API', () => {
     const projectId = 2;
     const groupId = 23;
     const reportId = '123';
-    const reportName = 'costs-estimation';
+    const reportName = 'costs-estimation-pe';
```

Comment:
It's probably better to use the name `top-members`, since the schema below is for the top members report

Reply:
Done